lib/collection/variable-scope.js

```diff
@@ -82,9 +82,9 @@ _.inherit((
         // in above line, we clone the values if it is already a list. there is no point directly using the instance of
         // a variable list since one cannot be created with a parent reference to begin with.
 
-        this._layers = [];
-
         if (layers) {
+            this._layers = [];
```

Comment:
Looks like that was my fault 😓 
https://github.com/postmanlabs/postman-collection/pull/384/commits/851ca618f399a93926f56ba3fa7209e6e5473022